README.md

```diff
@@ -45,7 +45,22 @@ Welcome to the repository for the Hierarchical Engine for Large-scale Infrastruc
 
 **Motivation:** Energy systems and their associated information and communication technology systems are becoming increasingly intertwined. As a result, effectively designing, analyzing, and implementing modern energy systems increasingly relies on advanced modeling that simultaneously captures both the cyber and physical domains in combined simulations.  It is designed to increase scalability and portability in modeling advanced features of highly integrated power system and cyber-physical energy systems.
 
-## Installation Instructions
+## Installers
+
+### Windows
+
+Each [release](https://github.com/GMLC-TDC/HELICS/releases/tag/v2.1.1) generally comes with a Windows installer and a zip file.  The current version is the release with Visual Studio 2017 x64 and includes the binaries for matlab, python and Java.  For building with Debug support it is generally recommended to build from Source or use HELICS as a subproject.  We hope to automate this in the future to be able to automatically generate other installers with different visual studio versions
```

Comment:
Use this link instead: https://github.com/GMLC-TDC/HELICS/releases/latest

It redirects to the latest tag.